misc/msvs/vcxproj.template

```diff
@@ -34,9 +35,31 @@
   <ItemGroup Condition="'$(IncludeListImported)'==''">
     %%DEFAULT_ITEMS%%
   </ItemGroup>
-  <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
+  <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" Condition=" '$(PlatformToolset)' != 'Clang' "/>
   <ImportGroup Label="ExtensionTargets">
   </ImportGroup>
+
+  <Target Name="Build" Condition=" '$(PlatformToolset)' == 'Clang' ">
+    <Exec Command="scons vsproj=yes dev_build=yes" Condition="'$(OS)' != 'Windows_NT'"/>
+    <Exec Command="scons use_mingw=yes use_llvm=yes vsproj=yes dev_build=yes" Condition="'$(OS)' == 'Windows_NT'"/>
+  </Target>
+  <Target Name="Rebuild" DependsOnTargets="Clean;Build" Condition=" '$(PlatformToolset)' == 'Clang' "/>
+  <Target Name="Clean" Condition=" '$(PlatformToolset)' == 'Clang' ">
+    <Exec Command="scons --clean vsproj=yes dev_build=yes" Condition="'$(OS)' != 'Windows_NT'"/>
+    <Exec Command="scons --clean use_mingw=yes use_llvm=yes vsproj=yes dev_build=yes" Condition="'$(OS)' == 'Windows_NT'"/>
+  </Target>
```

Comment:
Each platform has their own recorded scons command line for build commands, so this is going to break every clang+windows-based platform build. I don't understand why these build commands are being hardcoded here?